ClientApp/KachnaOnline/src/app/states/states-list/states-list.component.html

```diff
@@ -0,0 +1,66 @@
+<h1>Historie a plán stavů</h1>
+<app-month-selection (monthChange)="monthChanged($event)" [justifyCenter]="false"></app-month-selection>
+<div class="mb-2">
+  <app-togglable-button text="Řadit od nejbližšího" (valueChanged)="sort($event)"
+                        [startingValue]="true"></app-togglable-button>
+</div>
+
+<h4 class="text-muted" *ngIf="hasError">Nepodařilo se načíst seznam stavů</h4>
+<h4 class="text-muted" *ngIf="!loaded">Načítám seznam stavů...</h4>
+
+<ng-container *ngIf="loaded && !hasError">
+  <h4 class="text-muted" *ngIf="states.length == 0 && isInFuture">V tomto měsíci nejsou naplánované žádné změny stavu
+    klubu.</h4>
+  <h4 class="text-muted" *ngIf="states.length == 0 && !isInFuture">V tomto měsíci neproběhly žádné změny stavu
+    klubu.</h4>
+
+  <div class="table-responsive-md" *ngIf="states.length != 0">
+    <table class="table table-striped">
```

Comment:
Please, apply the `clickable` class on the table

Reply:
Oh, never mind. For some reason I thought that the rows were clickable but it is only editable through the edit button.